src/client/Dockerfile

```diff
@@ -1,20 +1,27 @@
 # pull official base image
-FROM node:13.12.0-alpine
+FROM node:13.12.0-alpine as builder
 
 # set working directory
 WORKDIR /app
 
 # add `/app/node_modules/.bin` to $PATH
 ENV PATH /app/node_modules/.bin:$PATH
-
+# ENV PAWS_API_HOST paws-compose-server
 # install app dependencies
 COPY package.json ./
 COPY package-lock.json ./
 
 RUN npm install --silent
 
+COPY public ./public
+COPY src ./src
+
+RUN npm run build
```

Comment:
We should keep using `npm start` for now to get the dev build of React

Reply:
If we need to run build for ngnix than it makes sense. otherwise frontend developers shouldn't even be using the container to run the frontend, and in general they won't to run build, just npm install locally and then npm start:local (using our custom script in the package.json)